src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,34 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {/* <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">qui est esse</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <a className="UserInfo" href="mailto:Sincere@april.biz">
+            Leanne Graham
+          </a>
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae
+        ea dolores neque fugiat blanditiis voluptate porro vel nihil
+        molestiae ut reiciendis qui aperiam non debitis possimus qui neque
+        nisi nulla
+      </p>
+
+      <hr />
+
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    </div> */}
+
+    {posts.map(post => (
+      <PostInfo post={post} />
```

Comment:
Each child in a list should have a unique 'key' prop. When you render a list, React needs this key to identify each item in the list and maintain state consistency.